docs/sources/enterprise/enterprise-configuration.md

```diff
@@ -361,6 +361,19 @@ This value limits the size of a single cache value. If a cache value (or query r
 
 The default is `1`.
 
+## [caching.encryption]
+### enabled
+
+Setting 'enabled' to `true` allows users to encrypt the query values in the cache.
```

Comment:
```suggestion
When 'enabled' to `true`, this setting allows users to encrypt the query values in the cache. Default is `false`.
```

Reply:
I updated it to
```
When 'enabled' is `true`, query values in the cache are encrypted.

The default is `false`.
```

Hopefully this is a bit more clear, but let me know what you think